selfdrive/test/test_onroad.py

```diff
@@ -28,6 +28,7 @@
   "selfdrive.controls.controlsd": 39.0,
   "./loggerd": 14.0,
   "./encoderd": 17.0,
+  "stream_encoderd": 17.0,
```

Comment:
this shouldn't be necessary since this test runs as a car, see https://github.com/commaai/openpilot/blob/8076107b4fa1b3c2b7c14a8464aa92e949e2e139/selfdrive/test/test_onroad.py#L261C114-L261C114